utils/paths.go

```diff
@@ -35,10 +38,24 @@ func GetCurationCacheFolder() (string, error) {
 	return filepath.Join(curationFolder, "cache"), nil
 }
 
-func GetCurationMavenCacheFolder() (string, error) {
+func GetCurationMavenCacheFolder(withProjectDir bool) (string, error) {
```

Comment:
I can't see anywhere that we pass this func a 'false' value.
you can remove the parameter and change the func name to fit it better.

Reply:
yeah, actually its for future use when we will add an option to clear all cache, or for specific project, but yeah it can be removed until we will add it